sys/include/net/csma_sender.h

```diff
@@ -84,9 +85,8 @@ void set_csma_mac_max_csma_backoffs(uint8_t val);
  * CSMA/CA, this feature is used. Otherwise, a software procedure is used.
  *
  * @param[in] dev       netdev device, needs to be already initialized
- * @param[in] pkt       pointer to the data in the packet buffer;
- *                      it must be a complete 802.15.4-compliant frame,
```

Comment:
Codewise, I did not see why this was necessary so I removed this statement here and in the other functions. Please let me know if you agree.